NOTICE

```diff
@@ -1,4 +1,6 @@
 
+Some material is derived from openembedded.org and is licensed in the same manner here (MIT for metadata; source code to match the associated recipe license).
```

Comment:
do we need this

Reply:
Hi Chandra: One recipe is matching in the blackduck tool, so is not original to here.  It is good practice to give some credit.  As the original recipe did not have a header, we add this generic note to NOTICE. - Martin

Reply:
ok,thanks martin and sandhya